electron/electron.js

```diff
@@ -65,9 +65,9 @@ app.on('activate', () => {
 
 ipcMain.handle('get-config-schema', async () => configSchema);
 
-ipcMain.handle('get-file', async () =>
+ipcMain.handle('get-file', async (event, extensions) =>
   dialog.showOpenDialog(mainWindow, {
-    filters: [{ name: 'JSON', extensions: ['json'] }],
+    filters: [{ name: 'extensions', extensions }],
```

Comment:
Do you know what the `name` field is for? If it's a name relevant to the dialog itself, I'm not sure if `'extensions'` is an informative name. Maybe we don't need to add it at all?

Reply:
filters is an array of FileFilter objects. Each of the FileFilters has a name. I'm not sure what the name is actually for; I think so you can differentiate between the different objects? But the name isn't displayed anywhere, and deleting it makes the file filter stop working.